src/options.ts

```diff
@@ -90,4 +90,12 @@ export interface GtmSupportOptions {
    * @default content-view
    */
   trackViewEventProperty?: string;
+  /**
+   * The name of the dataLayer variable to push events.
+   *
+   * @example dataLayerName: 'dataLayerDemo'
+   *
+   * @default dataLayer
+   */
```

Comment:
we could add an `@see https://developers.google.com/tag-platform/tag-manager/datalayer#rename_the_data_layer` to this JSDoc :thinking: